### DISCUSSION TITLE
Node statistics

### DISCUSSION THREAD

**Discussion by Botched1** [zwave-js/zwave-js-ui#2144] [General]
Sorry if this has been asked - I didn't readily see it when searching.

Is there anywhere to go to see the connection statistics on a node? Specifically looking for the PER, RTT, RSSI, speed, (40kb, 100kb, etc), and Route Changes info, as it can be very useful in troubleshooting mesh or device issues.

**Comment by robertsLando** [comment]
@Botched1 ATM the only stats available are the ones you see in control panel, just hover with the mouse on up/down arrows of a specific row or expand a node row and go on statistics. Other info should be available on network graph but not all the ones you are asking here
